src/pages/IdeLoader/IdeLoader.styl

```diff
@@ -0,0 +1,46 @@
+.ide-loader-tabs
+  .pf-c-tab-content
+    outline none
+
+  .pf-c-tabs
+    padding-bottom 0
+
+.ide-loader-wizard .pf-c-wizard__nav
+  width 100%
+  border none
+  font-size large
```

Comment:
In accordance to the prototype, font size of wizard nav items should be default (16px) but adding this rule increases it to 18px.
```suggestion
```

Reply:
I have done